debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogFileCollectorTest.java

```diff
@@ -1810,6 +1811,104 @@ public void testOpenRedoThreadSwitchDoesNotMissArchiveLogIfNotYetAvailable() thr
         assertThat(collector.isLogFileListConsistent(Scn.valueOf(1234567250), files, redoThreadState)).isTrue();
     }
 
+    @Test
+    @FixFor("DBZ-8162")
+    public void testOpenRedoThreadAfterBeingDoesNotBecomeInconsistent() throws Exception {
+        // In this scenario, a node in the RAC cluster is taken down for a period of time duration a
+        // maintenance window. The connector and environment continue onward for some time and the
+        // DBA brings the node back online, at which point the open log check for SCN should not fail.
+        RedoThreadState redoThreadState = RedoThreadState.builder()
+                .thread()
+                .threadId(1)
+                .status("OPEN")
+                .enabled("PUBLIC")
+                .enabledScn(Scn.valueOf(1918873))
+                .checkpointScn(Scn.valueOf(14338976116101L))
+                .lastRedoScn(Scn.valueOf(14339038344586L))
+                .lastRedoSequenceNumber(164492L)
+                .build()
+                .thread()
+                .threadId(2)
+                .status("OPEN")
+                .enabled("PUBLIC")
+                .enabledScn(Scn.valueOf(2028329))
+                .checkpointScn(Scn.valueOf(14338976116789L))
+                .lastRedoScn(Scn.valueOf(14339038344484L))
+                .lastRedoSequenceNumber(138697L)
+                .build()
+                .build();
```

Comment:
why twice?

Reply:
The first call builds `RedoThread` and the second builds final `RedoThreadState`.